client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -155,29 +157,25 @@ const DesignSelector: FunctionComponent< Props > = ( { showPageSelector = false
 				</div>
 			</animated.div>
 
-			<Portal>
-				<DialogBackdrop
-					visible={ showPageSelector }
-					className="design-selector__page-layout-backdrop"
-				/>
-			</Portal>
-
-			<Dialog
-				{ ...dialog }
-				hide={ () => {
-					history.push( makePath( Step.DesignSelection ) );
-				} }
-				aria-labelledby="page-layout-selector__title"
-				hideOnClickOutside
-				hideOnEsc
+			<animated.div
+				className={ classnames( 'design-selector__page-layout-container', {
+					'is-open': showPageSelector,
+				} ) }
+				style={ pageSelectorSpring }
 			>
-				<animated.div
-					className="design-selector__page-layout-container"
-					style={ pageSelectorSpring }
+				<Dialog
+					{ ...dialog }
+					modal={ false }
+					hide={ () => {
+						history.push( makePath( Step.DesignSelection ) );
+					} }
+					aria-labelledby="page-layout-selector__title"
+					hideOnClickOutside={ false }
+					hideOnEsc
 				>
 					<PageLayoutSelector templates={ otherTemplates } />
-				</animated.div>
-			</Dialog>
+				</Dialog>
+			</animated.div>
```

Comment:
These entire block is to move the `<animated.div>` wrapper out from the `<PageLayoutSelector>` into `<Dialog>` to fix the enter/exit animation regression.